cmd/install.go

```diff
@@ -45,43 +51,63 @@ And to see options for a specific app before installing, run:
+		"istio":                   NewArkadeApp("istio", apps.MakeInstallIstio, apps.IstioInfoMsg),
+		"crossplane":              NewArkadeApp("crossplane", apps.MakeInstallCrossplane, apps.CrossplainInfoMsg),
+		"docker-registry-ingress": NewArkadeApp("docker-registry-ingress", apps.MakeInstallRegistryIngress, apps.RegistryIngressInfoMsg),
+		"postgresql":              NewArkadeApp("postgresql", apps.MakeInstallPostgresql, apps.PostgresqlInfoMsg),
+		"minio":                   NewArkadeApp("minio", apps.MakeInstallMinio, apps.MinioInfoMsg),
+		"openfaas":                NewArkadeApp("openfaas", apps.MakeInstallOpenFaaS, apps.OpenFaaSInfoMsg),
+		"ingress-nginx":           NewArkadeApp("ingress-nginx", apps.MakeInstallNginx, apps.NginxIngressInfoMsg),
+		"nginx-ingress":           NewArkadeApp("nginx-ingress", apps.MakeInstallNginx, apps.NginxIngressInfoMsg), // backward compatability
+		"cert-manager":            NewArkadeApp("cert-manager", apps.MakeInstallCertManager, apps.CertManagerInfoMsg),
+		"openfaas-ingress":        NewArkadeApp("openfaas-ingress", apps.MakeInstallOpenFaaSIngress, apps.OpenfaasIngressInfoMsg),
+		"openfaas-loki":           NewArkadeApp("openfaas-loki", apps.MakeInstallOpenFaaSLoki, apps.LokiOFInfoMsg),
+		"loki":                    NewArkadeApp("loki", apps.MakeInstallLoki, apps.LokiInfoMsg),
+		"redis":                   NewArkadeApp("redis", apps.MakeInstallRedis, apps.RedisInfoMsg),
+		"nats-connector":          NewArkadeApp("nats-connector", apps.MakeInstallNATSConnector, apps.NATSConnectorInfoMsg),
+		"jenkins":                 NewArkadeApp("jenkins", apps.MakeInstallJenkins, apps.JenkinsInfoMsg),
+		"portainer":               NewArkadeApp("portainer", apps.MakeInstallPortainer, apps.PortainerInfoMsg),
+		"argocd":                  NewArkadeApp("argocd", apps.MakeInstallArgoCD, apps.ArgoCDInfoMsg),
+		"grafana":                 NewArkadeApp("grafana", apps.MakeInstallGrafana, apps.GrafanaInfoMsg),
+		"tekton":                  NewArkadeApp("tekton", apps.MakeInstallTekton, apps.TektonInfoMsg),
+		"traefik2":                NewArkadeApp("traefik2", apps.MakeInstallTraefik2, apps.Traefik2InfoMsg),
+		"inlets-operator":         NewArkadeApp("inlets-operator", apps.MakeInstallInletsOperator, apps.InletsOperatorInfoMsg),
+		"nfs-provisioner":         NewArkadeApp("nfs-client-provisioner", apps.MakeInstallNfsProvisioner, apps.NfsClientProvisioneriInfoMsg),
+		"docker-registry":         NewArkadeApp("docker-registry", apps.MakeInstallRegistry, apps.RegistryInfoMsg),
+		"OSM":                     NewArkadeApp("OSM", apps.MakeInstallOSM, apps.OSMInfoMsg),
+		"kube-image-prefetch":     NewArkadeApp("kube-image-prefetch", apps.MakeInstallKubeImagePrefetch, apps.KubeImagePrefetchInfoMsg),
+		"registry-creds":          NewArkadeApp("registry-creds", apps.MakeInstallRegistryCredsOperator, apps.RegistryCredsOperatorInfoMsg),
+		"gitea":                   NewArkadeApp("gitea", apps.MakeInstallGitea, apps.GiteaInfoMsg),
```

Comment:
I'm seeing the same string duplicated twice on each line - once in the map and one in the function invocation.

Reply:
yeh i did think about that, ill see if i can work out a better way